stable/prometheus-cloudwatch-exporter/templates/deployment.yaml

```diff
@@ -49,11 +43,11 @@ spec:
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           ports:
             - name: {{ .Values.service.portName }}
-              containerPort: {{ .Values.service.targetPort }}
+              containerPort: 9106
```

Comment:
The port specified by the entrypoint in the Docker image is 9106 as per https://github.com/prometheus/cloudwatch_exporter/blob/master/Dockerfile#L17